internal/agent/hooks/kcrypt_test.go

```diff
@@ -0,0 +1,102 @@
+package hook
+
+import (
+	"io/ioutil"
+	"os"
+	"testing"
+)
+
+func TestParsePartitionInfoFile(t *testing.T) {
```

Comment:
I'm totally fine with this, but just wondering why not using ginkgo also here?

Reply:
Since ginkgo wasn't there already, I thought it would be simpler without it. Turns out ginkgo tests are a lot more readable :D . I will switch to ginkgo for consistency across the projects (and for readability)

Reply:
Migrated to ginkgo here: https://github.com/kairos-io/kcrypt/pull/3/commits/82c6e8fcd092bd05aeda8fd5ef292965957ee95f